internal/server/everest.go

```diff
@@ -294,8 +302,34 @@ func newSkipperFunc() (echomiddleware.Skipper, error) {
 }
 
 // Start starts everest server.
-func (e *EverestServer) Start() error {
-	return e.echo.Start(fmt.Sprintf("0.0.0.0:%d", e.config.HTTPPort))
+func (e *EverestServer) Start(ctx context.Context) error {
+	addr := fmt.Sprintf("0.0.0.0:%d", e.config.ListenPort)
+	if e.config.TLSCertsPath != "" {
+		return e.startHTTPS(ctx, addr)
+	}
+	return e.echo.Start(addr)
+}
+
+func (e *EverestServer) startHTTPS(ctx context.Context, addr string) error {
+	tlsKeyPath := path.Join(e.config.TLSCertsPath, "tls.key")
+	tlsCertPath := path.Join(e.config.TLSCertsPath, "tls.crt")
+
```

Comment:
```suggestion
	tlsKeyPath := path.Join(filepath.Clean(e.config.TLSCertsPath), "tls.key")
	tlsCertPath := path.Join(filepath.Clean(e.config.TLSCertsPath), "tls.crt")

```

Reply:
or even better - apply cleaning during reading the value from config. In such a case you always refer to already cleaned path each time you need to get this value from config obj